CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## 2015-02-19
```

Comment:
Thanks for adding a Changelog. Good to have. Could you change this line to:

`# 3.0.2`